Marlin/src/feature/pause.cpp

```diff
@@ -143,13 +143,14 @@ bool pause_print(const float &retract, const float &z_lift, const float &x_pos,
   // Save current position
   stepper.synchronize();
   COPY(resume_position, current_position);
-
+  set_current_to_destination();            // Reset all old destinations to current
```

Comment:
**RUNPLAN Vs Doblockmove**
RUNPLAN need ' destinations ' to move and after you manually set ' current pos '
Doblockmoves apply directly the new current position , and ask planner to go there ' I LOVE THIS WAY'
For these reason , we are forced to prepare ' current ' and ' destination ' before each
In this case ' retract is a RUNPLAN , then we ensure the current pos and we set destination and because of a doblockmove just after we have to apply the actual current value when finish '